### PR TITLE
DeadCodeCleanup: Rules

### PR DESCRIPTION
**Pull request by MarcusDenker**
Small dead code cleanup in Renraku and GeneralRules

- remove old deprecated methods
- remove unsent private methods

### CODE REVIEW

File: src/Renraku/RBTransformationRule.extension.st [hunk 1]
```diff
@@ -12,13 +12,3 @@ RBTransformationRule >> check: aMethod forCritiquesDo: aCriticBlock [
 					tree: rewriteRule tree)  ]
 		
 ]
-
```
Comment (Ducasse) [inline src/Renraku/RBTransformationRule.extension.st]: Marcus I think that this one is nice to keep.

Comment (Ducasse) [reply src/Renraku/RBTransformationRule.extension.st]: Talking about suggestedCode:

Comment (MarcusDenker) [reply src/Renraku/RBTransformationRule.extension.st]: My problem is that I deleted the image already and I still do no know how to load this PR now and update it...

**Comment by Ducasse** [comment]
Check your remote and it should contain all your branches. 
And you can checkout that branch. 
And this is a nice XP to do.

**Comment by Ducasse** [comment]
Thanks!

**Pull request merged**